apps/chronicles/mad.config.ts

```diff
@@ -17,4 +20,11 @@ export const config: MadConfig = {
         title: "Chronicles",
         logo: Logo,
     },
+    serviceNow: {
+        whatever: "",
```

Comment:
This looks like a placeholder? Maybe add a todo if you are unable to implement this at the moment?

Reply:
Edit: I noticed you added a todo to the type declaration so looks like youre covered 👍